composites/Plugin/Shared/components/Button.js

```diff
@@ -268,15 +268,23 @@ IconButton.defaultProps = {
  * @returns {ReactElement} Styled icon button.
  */
 export const IconsButton = ( props ) => {
-	const { children: text, prefixIcon, prefixIconColor, suffixIcon, suffixIconColor } = props;
-
-	const newProps = omit( props, [ "prefixIcon", "prefixIconColor", "suffixIcon", "suffixIconColor" ] );
+	const {
```

Comment:
I don't think we want to pass all these props to the button?
I suggest using object destructuring.
```
const {
    extractedProp1,
    extractedProp2,
    ...buttonProps
} = props;
```

Reply:
Also, what a godawful amount of props. Consider grouping prefix and suffix props, so you get `props.suffix.icon`, `props.suffix.iconColor`, and so on. I think this improved readability overall.